### PR TITLE
travis: Enable ARM support in travis

### LINKED ISSUE #19982: Add ARM/ARM64 CI environments
Apparently there's a `big.Int` error in Go on ARM64 prior to 1.12.9 (https://github.com/golang/go/issues/32940), which may or may not be the only issue causing bad blocks (https://github.com/ethereum/go-ethereum/issues/19754).thttps://github.com/ethereum/go-ethereum/issues/19754). As ARM is getting more and more prevalent, I think we should consider expanding our CI with ARM targets too somehow.

### PR DESCRIPTION
**Pull request by gballet**
Travis added support for ARM during my holiday, so this PR activates the ARM build. This replaces #20039 

Closes #19982

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: go
 go_import_path: github.com/ethereum/go-ethereum
 sudo: false
+arch:
+  - amd64
+  - arch64
```
Comment (karalabe) [inline .travis.yml]: These two entries look like they mess up the pipeline (a `Test` section appeared): https://travis-ci.org/ethereum/go-ethereum/builds/604822629?utm_medium=notification&utm_source=github_status

File: metrics/doc.go [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package metrics
+
+const Epsilon = 0.0000000000000001
```
Comment (fjl) [inline metrics/doc.go]: please unexport

**Comment by fjl** [comment]
I think we should just bump the timeouts on all platforms.

**Comment by gballet** [comment]
@fjl I have applied all your review requests, PTAL.

Regarding the problem with the lookup, it seems that from time to time, the ARM VM has some issue contacting the DNS server. I have been able to reproduce the same issue with `ping` and therefore the fix I propose is to retry a couple times.

**Pull request merged**